src/discordcr/mappings/channel.cr

```diff
@@ -32,16 +50,23 @@ module Discord
     property member : PartialGuildMember?
     @[JSON::Field(converter: Discord::TimestampConverter)]
     property timestamp : Time
+    property edited_timestamp : Time?
     property tts : Bool
     property mention_everyone : Bool
     property mentions : Array(User)
     property mention_roles : Array(Snowflake)
+    property mention_channels : Array(Snowflake)?
     property attachments : Array(Attachment)
     property embeds : Array(Embed)
     property pinned : Bool?
     property reactions : Array(Reaction)?
     property nonce : String | Int64?
+    property webhook_id : Snowflake?
     property activity : Activity?
+    property application : Application?
+    property message_reference : MessageReference?
+    property flags : MessageFlags?
+    property referenced_message : JSON::Any?
```

Comment:
Would like to note that there are actually 3 states for this field:
1. Message object
2. null: message was deleted
3. undefined: wasn't fetched

> If the message is a reply but the referenced_message field is not present, the backend did not attempt to fetch
https://discord.com/developers/docs/resources/channel#message-object